### ISSUE TITLE
Using MATLAB within multiple threads

### ISSUE THREAD

**Issue by yakir12** [JuliaInterop/MATLAB.jl#169]
I get read errors or stalling if I try to use MATLAB with multiple threads. If this is indeed impossible, maybe mention this in the README/docs.

MWE:
```julia
using MATLAB, Base.Threads
@threads for _ in 1:10
    mat"1 + 1"
end
```

**Comment by musm** [comment]
Yeah this is an untested configuration that's unlikely to work without some help.

**Comment by VoSiLk** [comment]
I also had interferences in the values of some variables of scripts, which using MATLAB when they are calculating at the same time
